actionpack/test/controller/rescue_test.rb

```diff
@@ -306,6 +314,23 @@ def test_block_rescue_handler_with_argument_as_string
     get :exception_with_no_handler_for_wrapper
     assert_response :unprocessable_entity
   end
+
+  test "can rescue a ParserError" do
+    capture_log_output do
+      post :arbitrary_action, body: "{", as: :json
+    end
+    assert_response :ok
+    assert_equal "arbitrary action", response.body
```

Comment:
Should not the response be `"parser error"`?

Reply:
Whoops, I'll fix that, and look into why its failing.

Reply:
I didn't realize this at first, but it only blows up if you reference `params` in the controller. Is that acceptable or should we look at rescuing on the `request.filtered_parameters` call in instrumentation's `#process_action`?

Reply:
That is the expected behavior for me.